packages/gatsby-source-wordpress/src/steps/source-nodes/create-nodes/process-node.js

```diff
@@ -742,8 +742,8 @@ const replaceFileLinks = async ({
         if (node.internal.type === `File`) {
           fileNode = node
           mediaItemNode = await helpers.getNode(node.parent)
-        } else if (node.localFile?.id) {
-          fileNode = await helpers.getNode(node.localFile.id)
+        } else if (node.localFile?.id || typeof node.localFile === `string`) {
```

Comment:
Can we track down where node.localFile is being set and set it on node.localFile.id instead? I think the bug is that we're setting the id wrong (onto localFile instead of on localFile.id)